test/lib/ex_aws/ec2/integration_test.exs

```diff
@@ -5,4 +5,21 @@ defmodule ExAws.EC2.IntegrationTest do
     assert {:ok, %{body: body}} = ExAws.EC2.describe_instances |> ExAws.request
     assert body |> String.contains?("DescribeInstancesResponse")
   end
+
+  test "#describe_tags" do
```

Comment:
I used these for my own sanity testing, but they could potentially leak data (if the request succeeds but the pattern match fails) through travis CI if they were given describe on a * resource - I'm not sure if those tests are isolated in their own AWS account.